<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pqExampleVisualizationsDialog</class>
 <widget class="QDialog" name="pqExampleVisualizationsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>996</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TTK Example Visualizations</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0" colspan="3">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Click on one of thumbnails below to load an example visualization&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="titleLabel">
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <italic>false</italic>
       <strikeout>false</strikeout>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="text">
      <string>TTK Example Visualizations</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLabel" name="Example3Description">
     <property name="text">
      <string>Uncertain data example</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignHCenter|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="Example2Description">
     <property name="text">
      <string>Bivariate scalar data example</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignHCenter|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="Example3Button">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../pqApplicationComponents.qrc">
       <normaloff>:/pqApplicationComponents/Thumbnails/Example3.png</normaloff>:/pqApplicationComponents/Thumbnails/Example3.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>300</width>
       <height>225</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="Example1Description">
     <property name="text">
      <string>Scalar data example</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignHCenter|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="Example2Button">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normalon>:/pqApplicationComponents/Thumbnails/Example2.png</normalon>
      </iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>300</width>
       <height>225</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="Example1Button">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normalon>:/pqApplicationComponents/Thumbnails/Example1.png</normalon>
      </iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>300</width>
       <height>225</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>Example1Button</tabstop>
  <tabstop>Example2Button</tabstop>
  <tabstop>Example3Button</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="../pqApplicationComponents.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>pqExampleVisualizationsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>500</x>
     <y>348</y>
    </hint>
    <hint type="destinationlabel">
     <x>500</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>pqExampleVisualizationsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>500</x>
     <y>348</y>
    </hint>
    <hint type="destinationlabel">
     <x>500</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
